resources/[system]/chat/cl_chat.lua

```diff
@@ -15,18 +15,20 @@ RegisterNetEvent('_chat:messageEntered')
 
 --deprecated, use chat:addMessage

```

Comment:
\>deprecated
\>doesn't handle other chat APIs

needs a more consistent API and you can't even be sure of the order in which events execute so there's a chance your custom code canceling it runs after this...

... should just trigger another event in each of the add ones and check if _that_ was canceled.